app/subscriber/src/components/sidebar/advanced-search/AdvancedSearch.tsx

```diff
@@ -87,8 +88,9 @@ export const AdvancedSearch: React.FC<IAdvancedSearchProps> = ({ expanded, setEx
           Object.values(advancedSearch.searchInField).every((v) => v === false)
             ? advancedSearch.searchTerm
             : '',
+        productIds: advancedSearch?.frontPage ? [Number(process.env.FRONT_PAGE_ID)] : [],
```

Comment:
@Fosol - is this what you meant?

Reply:
You specific implementation won't work. Env values are not available to clients, only to the server.  For now a simple solution is to create a config JSON file somewhere with constants that we can include in the build of the image.  However, this has issues when the image is used with a different dataset.

Reply:
You can add a JSON file as constants, and then that file can be overridden by our build process if required.